src/bot.rs

```diff
@@ -143,6 +143,17 @@ impl Bot {
     /// Creates genesis group and qr-codes.
     /// Returns the complete bot config.
     async fn setup(context: &Context) -> Result<BotConfig> {
+        context
+            .set_config(
+                Config::Selfavatar,
+                Some(
+                    get_icon_path()?
+                        .to_str()
+                        .context("Can't convert image file")?,
+                ),
+            )
```

Comment:
Maybe use this opportunity to set a display name like "appstore: xstore@testrun.org"? But not a blocker, that's nice enough already.